website/converter_frontend/views.py

```diff
@@ -20,14 +21,32 @@ def contacts(request):
 
 
 def converter(request):
-
     path = os.path.join((os.getcwd()), 'data_file.json')
 
+    # first check if we have datafile, and download if it is not present
+    # ignore errors if any
+    if not os.path.isfile(path):
+        try:
+            os.system(f'python parser.py --path {os.getcwd()}')
+        except OSError:
```

Comment:
maybe we will not limit?

Reply:
As we've discussed via live chat, this single exception is enough.